lib/signs_ui/config/expiration.ex

```diff
@@ -47,22 +56,26 @@ defmodule SignsUi.Config.Expiration do
     {:noreply, state}
   end
 
-  @spec expire_signs(SignsUi.Config.State.t(), (() -> DateTime.t())) :: %{
+  @spec expire_signs(SignsUi.Config.State.t(), (() -> DateTime.t()), (() -> String.t())) :: %{
```

Comment:
The third argument here, the `alert_fetcher` is a function that returns a mapset of alert IDs, so it should have a spec of `(() -> MapSet.t(SignsUi.Alerts.Alert.id()))`.